molgenis-api-permissions/src/main/java/org/molgenis/api/permissions/model/request/TypePermissionsRequest.java

```diff
@@ -0,0 +1,20 @@
+package org.molgenis.api.permissions.model.request;
+
+import com.google.auto.value.AutoValue;
+import java.util.List;
+import org.molgenis.util.AutoGson;
+
+@AutoValue
+@AutoGson(autoValueClass = AutoValue_TypePermissionsRequest.class)
+@SuppressWarnings(
+    "squid:S1610") // Abstract classes without fields should be converted to interfaces
+public abstract class TypePermissionsRequest {
+  public abstract String getTypeId();
+
+  public abstract List<ObjectPermissionsRequest> getObjects();
```

Comment:
Is this list allowed to be empty?

Reply:
Fixed, commented line not changed so not marked as outdated